src/Components/Footer/MailingList/index.js

```diff
@@ -0,0 +1,16 @@
+import React, { Component } from 'react';
+
+import './index.css';
+import Input from '../../Input';
+
+
+export default class MailingList extends Component {
+  render() {
+    return (
+      <form className="Newsletter">
```

Comment:
I think it would be nicer if the className matched the component name. This way is easier to look up the files where the css is defined. What do you think?

Reply:
oh right! My bad. I'll change it to reflect the same names. :)